File: C#/Personne/Personne/Personne.cs
using System;

namespace Personne
{
    public class Personne
    {
        public string nom { get; set; }
        public string prenom { get; set; }
        public DateTime dateNaiss { get; set; }

        public Personne(string nom, string prenom, DateTime dateNaiss)
        {
            this.nom = nom;
            this.prenom = prenom;
            this.dateNaiss = dateNaiss;
        }

        public int age()
        {
            // Calculer la différence entre la date d'aujourd'hui et la date de naissance
            TimeSpan ageTimeSpan = DateTime.Now - dateNaiss;

            // Convertir la différence en jours
            int ageEnJours = ageTimeSpan.Days;

            // Diviser le nombre de jours par 365.25 pour obtenir l'âge en années
            int ageEnAnnees = (int)Math.Floor(ageEnJours / 365.25);

            // Retourner l'âge en années
            return ageEnAnnees;
        }

        public virtual void afficher()
        {
            Console.Write("Nom : " + this.nom + ", Prenom : " + this.prenom + ", Age : " + this.age());
        }
    }
}

File: Java/Classes/Personne.java
public class Personne{ 
 // attributs
 private String prenom; private String nom; 
 private int age; 
 
 // constructeurs
 public Personne(String P, String N, int age){ 
 initialise(P,N,age); 
 } 
 public Personne(Personne P){ 
 initialise(P); 
 } 
 // méthode
 public void initialise(String P, String N, int age){ 
 this.prenom=P; 
 this.nom=N; this.age=age; 
 } 
 public void initialise(Personne P){ this.prenom=P.prenom; 
 this.nom=P.nom; 
 this.age=P.age; 
 } 
 // méthode
 public void identifie(){ 
 System.out.println(prenom+","+nom+","+age); 
 } 
}